.github/scripts/tryrebase.py

```diff
@@ -48,6 +48,14 @@ def rebase_ghstack_onto(pr: GitHubPR, repo: GitRepo, dry_run: bool = False, stab
 
     repo.fetch(orig_ref, orig_ref)
     repo._run_git("rebase", onto_branch, orig_ref)
+
+    os.environ["OAUTH_TOKEN"] = os.environ["GITHUB_TOKEN"]
+    with open('.ghstackrc', 'w+') as f:
+        f.write('[ghstack]\n' +
+                "github_url=github.com\n" +
+                "github_username=clee2000\n" +
```

Comment:
Should this always be you?

Reply:
one days, im going to remember to change all of these